specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-11-01-preview/keys.json

```diff
@@ -0,0 +1,670 @@
+            "enum": [
+              "encrypt",
+              "decrypt",
+              "sign",
+              "verify",
+              "wrapKey",
+              "unwrapKey",
+              "import",
+              "release"
+            ],
+            "x-ms-enum": {
+              "name": "JsonWebKeyOperation",
+              "modelAsString": true
+            }
+          }
+        },
+        "keySize": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The key size in bits. For example: 2048, 3072, or 4096 for RSA."
+        },
+        "curveName": {
+          "type": "string",
+          "description": "The elliptic curve name. For valid values, see JsonWebKeyCurveName.",
+          "enum": [
+            "P-256",
+            "P-384",
+            "P-521",
+            "P-256K"
+          ],
+          "x-ms-enum": {
+            "name": "JsonWebKeyCurveName",
+            "modelAsString": true
+          }
+        },
+        "keyUri": {
+          "type": "string",
+          "description": "The URI to retrieve the current version of the key.",
+          "readOnly": true
+        },
+        "keyUriWithVersion": {
+          "type": "string",
+          "description": "The URI to retrieve the specific version of the key.",
+          "readOnly": true
+        },
+        "rotationPolicy": {
+          "$ref": "#/definitions/RotationPolicy",
+          "description": "Key rotation policy in response. It will be used for both output and input. Omitted if empty"
+        },
+        "release_policy": {
+          "$ref": "#/definitions/KeyReleasePolicy",
+          "description": "Key release policy in response. It will be used for both output and input. Omitted if empty"
+        }
+      },
+      "description": "The properties of the key.",
+      "type": "object"
+    },
+    "KeyAttributes": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Determines whether or not the object is enabled."
+        },
+        "nbf": {
+          "x-ms-client-name": "NotBefore",
+          "type": "integer",
+          "format": "int64",
+          "description": "Not before date in seconds since 1970-01-01T00:00:00Z."
+        },
+        "exp": {
+          "x-ms-client-name": "Expires",
+          "type": "integer",
+          "format": "int64",
+          "description": "Expiry date in seconds since 1970-01-01T00:00:00Z."
+        },
+        "created": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "Creation time in seconds since 1970-01-01T00:00:00Z."
+        },
+        "updated": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "Last updated time in seconds since 1970-01-01T00:00:00Z."
+        },
```

Comment:
The format should be `unixtime` unless that would be considered breaking. See https://github.com/Azure/azure-rest-api-specs/blob/5e455f9123f2c0c7fe22f3e43e0402ae8cf8e807/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.2/common.json#L10-L42 for what the data plane uses. In fact, assuming that secrets.json also has these attributes, like the data plane, I recommend putting these in common.json and sharing them via an `allOf` reference like we do in the data plane.

Reply:
unixtime was indeed considered a breaking change, there was an old email thread about this. I'll see if I can dig it up

Reply:
The numeric representation should be the same anyway, or was it an issue because client code was generated differently?

Reply:
I see they were int64, already. Unfortunate, but I still think we should investigate fixing this because the numeric representation is the same.

https://github.com/Azure/azure-rest-api-specs/blob/5e455f9123f2c0c7fe22f3e43e0402ae8cf8e807/specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/keys.json#L382-L405

I'll change my PR vote, but I'd like to see a subsequent PR to use the same CloudError eventually from managedHsm.json.